src/crypto/x509.js

```diff
@@ -186,6 +186,11 @@ export default class X509Chain {
         let signerCert = this.certChain[0];
         return signerCert.subjectPublicKeyInfo;
     }
+
+    getParameterValueByName(parameterName) {
+        let signerCert = this.certChain[0];
+        return signerCert[parameterName]?.typesAndValues[0]?.value?.valueBlock?.value || '';
```

Comment:
Maybe we should return null in case parameter is missing, because probably empty value is normal value for some of the parameters and we will need to distinguish them in future.